src/Constants/commands_messages.py

```diff
@@ -1,12 +1,17 @@
 CANCEL_ACTION_MESSAGE = "Canceling action: {}"
 CREATE_FOOD_STORAGE_ACTION = "Create {} with name '{}'."
-CREATE_FOOD_STORAGE_NAME_MESSAGE = "What would like to name this {}?: "
+CREATE_FOOD_STORAGE_NAME_MESSAGE = "What would you like to name this {}?: "
 CREATE_FOOD_STORAGE_SUCCESS_MESSAGE = "Successfully created {}! Now opening: {}"
 CREATE_PROFILE_NAME_MESSAGE = "What would you like to name your profile?: "
 CREATE_PROFILE_SUCCESS_MESSAGE = "Successfully created profile. Now switching to profile: {}"
 EXISTING_NAME_MESSAGE = "A {} with the name '{}' already exists."
 INVALID_RESPONSE_MESSAGE = "Sorry, that response was not recognized. Please try again."
+NEW_FOOD_STORAGE_REQUEST_MESSAGE = "This food storage type does not exist. Would you like to create one? (y/n): "
 NO_LOADED_PROFILE_MESSAGE = "Please create or load a profile before proceeding."
+OPEN_FOOD_STORAGE_ACTION = "Opening food storage."
+OPEN_FOOD_STORAGE_MESSAGE = "You would like to open a food storage. Here are the food storages available: {}"
```

Comment:
There should probably also be a constant for if there are no foodStorages and another for saying like please create one using whatever command

Reply:
I actually did something like this in my PR that's not done that you can kinda look at in [session.py](https://github.com/amandate/fridge/pull/20/files)

You can create a Utils package (a Utils Folder with a `__init__.py` file in it like the others have) and basically kinda copy the `_printSavedProfiles` from my PR into it. Change it slightly to be more generic and have it take in a parameter of the things it would be printing. This way we can use it for both situations.

Reply:
I would say focus on this part last because I think I was actually almost done with my PR so I can do it if you want.

Reply:
I ended up handling creating the print method in my PR so you don't have to worry about that!